### PR TITLE
Changed message for opening link in new tab dialog

### LINKED ISSUE #1941: Bug in open link in new tab dialog
**Describe the bug**
Message in open link in new tab is written twice.

**Expected behavior**
Message should be written only once.

**Steps to reproduce the behavior:**
1. Open any Zim file.
2. Long click on a highlighted work.
3. Dialog box appears with "Open link in new tab option" written twice.

**Screenshots**
Screenshot_2020-03-21-20-56-06-096_org kiwix kiwixmobile

### PR DESCRIPTION
**Pull request by ashutosh1919**
Fixes #1941 .
@macgills and @kelson42, Please review.

**Screenshots** 
PHOTO-2020-03-24-12-24-58

### CODE REVIEW

**Comment by yashk2000** [comment]
@ashutosh1919 this issue was already assigned to @yash1ts who is currently working on it. You should ask the current person who is working on the issue before sending a pr if someone is already working on it.

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,206 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
```
Comment (yashk2000) [inline .idea/codeStyles/Project.xml]: Please make sure that when you are sending a pr that the .idea files are not committed. Only the files you made changes to should be committed and pushed.

Comment (ashutosh1919) [reply .idea/codeStyles/Project.xml]: Yes I am changing it. Give me half an hour.

Comment (yashk2000) [reply .idea/codeStyles/Project.xml]: Read my comment above first. Before you continue working on this, ask @yash1ts as the issue is assigned to him. If he is working on it, there is no point in you sending a pr.

Comment (ashutosh1919) [reply .idea/codeStyles/Project.xml]: @yashk2000 , I have commented on the original issue.

File: core/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -133,6 +133,7 @@
   <string name="external_storage">External</string>
   <string name="yes">Yes</string>
   <string name="no">No</string>
+  <string name="confirm_open_in_new_tab">Are you sure you want to open link in new tab?</string>
```
Comment (rashiq) [inline core/src/main/res/values/strings.xml]: The copy makes you think you're doing something wrong by opening the link in a new tab. 

> Do you want to open the link in a new tab?

imo sounds better

Comment (ashutosh1919) [reply core/src/main/res/values/strings.xml]: Done. @rashiq , please take a look at it now.

File: core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt [hunk 3]
```diff
@@ -178,7 +178,7 @@ sealed class KiwixDialog(
     )
 
     object OpenInNewTab : YesNoDialog(
-      R.string.open_in_new_tab, R.string.open_in_new_tab
+      R.string.open_in_new_tab, R.string.confirm_open_in_new_tab
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt]: set the title to null and use `open_in_new_tab` for the message, this dialog is too redundant

Comment (ashutosh1919) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt]: done

**Comment by ashutosh1919** [comment]
@rashiq and @macgills , Changes done.

**Comment by macgills** [comment]
Right, I had forgotten which one we had made nullable. just flip the parameters

**Comment by ashutosh1919** [comment]
@macgills, I have checked with the both params. We can’t keep anyone of them null.

**Comment by macgills** [comment]
That is a limitation of this subclass,
```
sealed class KiwixDialog(
  val title: Int?,
  val message: Int?,
  val positiveMessage: Int,
  val negativeMessage: Int?,
  val cancelable: Boolean = true,
  val icon: Int? = null,
  val neutralMessage: Int? = null,
  val getView: (() -> View)? = null
) {
```
Pretty much everything is nullable in the base class

Just null these up 
```
open class YesNoDialog(
    title: Int,
    message: Int
  ) 
```

Setting title null seems to be what we do for most other dialogs so do that

**Comment by ashutosh1919** [comment]
Okay i got it. So, let’s keep title null then.

**Comment by macgills** [comment]
make it nullable
```
open class YesNoDialog(
    title: Int?,
    message: Int
  ) 
```

You should also really be testing every commit. Seeing as this is a compile time error that would be impossible, the IDE should also have a lot of red highlighting. `alt+enter` will probably suggest an autofix even

**Comment by ashutosh1919** [comment]
@macgills and @rashiq , below attached screenshot is of latest commit. Please review.
PHOTO-2020-03-26-12-25-49

**Pull request merged**